package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pg-txclient",
-  "version": "0.0.9",
+  "version": "0.1.0",
```

Comment:
It's unexpected that the `"0.1.0"` get highlighted as a change, i don't see in master the `"0.0.9"` version. Let's keep  this as `"0.1.0"` to be consistent with the current master..